deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -474,6 +474,11 @@ <h4 class="panel-title pull-left">Capacity</h4>
                 return false
             }
 
+            if ('ebs_volume_type' in clusterInfo.configs && (clusterInfo.configs['ebs_volume_type'] != "gp2" && clusterInfo.configs['ebs_volume_type'] != "gp3")) {
```

Comment:
Will this value be applied to both root and non-root volumes?

Reply:
only for root_volume_type for now as I am not aware what volume types are supported for ebs_volume_type